.gitignore

```diff
@@ -1,3 +1,5 @@
 manifests/deis-router-rc.tmp.yaml
+rootfs/bin/router
```

Comment:
I don't think this addition is required.  What is most likely is that you have a vestigial binary remaining in this old location from older build.  We used to put it there... but then it moved to its present location.